refinerycms-blog.gemspec

```diff
@@ -24,6 +24,6 @@ Gem::Specification.new do |s|
   s.add_dependency    'friendly_id',           ['< 5.3', '>= 5.1.0']
   s.add_dependency    'globalize',             '~> 5.1.0'
   s.add_dependency    'activemodel-serializers-xml', '~> 1.0', '>= 1.0.1'
-  s.add_dependency    'jquery-ui-rails',       '~> 5.0.0'
+  s.add_dependency    'jquery-ui-rails',       '~> 6.0.0'
```

Comment:
We could just remove this dependency because `refinerycms-core` has already this dependency:
https://github.com/refinery/refinerycms/search?q=jquery-ui-rails&unscoped_q=jquery-ui-rails